java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -980,18 +1002,27 @@ public static ColumnVector md5Hash(ColumnVector... columns) {
     if (columns.length < 1) {
       throw new IllegalArgumentException("MD5 hashing requires at least 1 column of input");
     }
-    long[] column_views = new long[columns.length];
+    long[] columnViews = new long[columns.length];
     long size = columns[0].getRowCount();
 
     for(int i = 0; i < columns.length; i++) {
       assert columns[i] != null : "Column vectors passed may not be null";
-      assert columns[i].getRowCount() == size : "Row count mismatch, all columns must have the same number of rows";
+      assert columns[i].getRowCount() == size : "Row count mismatch, all columns must be the same size";
       assert !columns[i].getType().isDurationType() : "Unsupported column type Duration";
       assert !columns[i].getType().isTimestamp() : "Unsupported column type Timestamp";
-      column_views[i] = columns[i].getNativeView();
+      if(columns[i].getType() == DType.LIST) {
+        assert columns[i].getNumChildren() == 1 : "Unrecognized list column format";
+        ColumnViewAccess listDataColumn = columns[i].getChildColumnViewAccess(0);
+        assert !listDataColumn.getDataType().isDurationType() : "Unsupported column type List of Durations";
+        assert !listDataColumn.getDataType().isTimestamp() : "Unsupported column type List of Timestamp";
+        assert !listDataColumn.getDataType().isNestedType() : "Unsupported column type List of nested types";
```

Comment:
nit: does the underlying hash code already test for these same assertions?  If so then we don't need to do it here.